package/src/components/DataTable/DataTable.js

```diff
@@ -0,0 +1,350 @@
+import React from "react";
+import PropTypes from "prop-types";
+import clsx from "clsx";
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TextField,
+  TableRow,
+  Toolbar,
+  makeStyles,
+  Typography
+} from "@material-ui/core";
+import ChevronLeftIcon from "mdi-material-ui/ChevronLeft";
+import ChevronRightIcon from "mdi-material-ui/ChevronRight";
+import Button from "../Button";
+import Select from "../Select";
+import ActionMenu from "../ActionMenu";
+
+const useStyles = makeStyles((theme) => ({
+  pagination: {
+    color: theme.palette.colors.coolGrey500,
+    letterSpacing: 0.28,
```

Comment:
You can use `captionLetterSpacing` theme variable here.

Reply:
I think this could be the `<Typography variant=body2 />` as its the best match (if not an exact match for the letter spacing, color, and font size)

Reply:
or that, too!